crates/uv/tests/pip_compile.rs

```diff
@@ -4573,6 +4573,13 @@ fn invalid_metadata_requires_python() -> Result<()> {
           unusable because the package metadata could not be parsed and you
           require validation==2.0.0, we can conclude that the requirements are
           unsatisfiable.
+
+          hint: Metadata for validation==2.0.0 could not be parsed:
+            Failed to parse version: Unexpected end of version specifier, expected
+          operator:
+            12
```

Comment:
I thought we turned off the wrapping here.

Reply:
Oh I see interesting... is `textwrap::indent` doing this?

Reply:
Oh maybe. I could disable it manually by reading the env var... We use it in a few places.

Reply:
We should probably implement https://github.com/zkat/miette/pull/328

Reply:
Oh actually, I don't think `textwrap::indent` does this. We're just missing `UV_NO_WRAP=1`. I'll put up a separate PR.

Reply:
Ah yeah miette used `textwrap::fill` instead of `indent`.